crates/solidity/outputs/cargo/slang_solidity/src/generated/bindings/generated/built_ins.rs

```diff
@@ -0,0 +1,58 @@
+// This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+use semver::Version;
+
+#[allow(unused_variables)]
+pub fn get_contents(version: &Version) -> &'static str {
+    if *version < Version::new(0, 5, 0) {
+        r####"contract $$ {
```

Comment:
suggestion instead of using `$$`, as this will be user visible.. would that work for our internal bindings?

```suggestion
        r####"interface $BuiltIn$Functions {
```

Reply:
Yes, we can use any name here. I had already changed the name in the complete built-ins branch. The important thing is keeping it in sync with the rules file where it's linked.

Using an interface wouldn't work, since the rules would not bind the declared state variables as globals, because state vars are not allowed in interfaces (even though they will parse successfully). We could make a special case in the rules file if needed though.

Reply:
> because state vars are not allowed in interfaces
> We could make a special case in the rules file

That makes sense. Let's stick to the language semantics, to make sure they are understood by users.
In that case, `contract $BuiltIns$` sounds good to me (PascalCase).